### ISSUE TITLE
Version Command Ouputs Useless Number

### ISSUE THREAD

**Issue by tmob03** [ValveSoftware/Source-1-Games#4739]
Ever since SDK 2013, the `version` command outputs a useless number
![image](https://user-images.githubusercontent.com/33987871/213948547-33a50336-edd9-4fd6-bcc8-00368ecca9e6.png)

Before 2013, it used to output the number of days since November 7, 1998 and thus was a useful number. The larger the number, the newer the build. With how it is now, a bigger number means absolutely nothing. If we had the source code, it wouldn't be useless unless it's really just a random number.

**Comment by Thespikedballofdoom** [comment]
Doesn't a bigger number mean a newer patch still?

**Comment by tmob03** [comment]
> Doesn't a bigger number mean a newer patch still?

Yes, but it gives no info on the date or anything. I didn't realize that when making this issue.

I have tried figuring out the increase per date, but it doesn't seem to be linear at all, with old versions' version numbers rising more slowly than recent.

The last two HL2 client updates had their version numbers rise 203,814 over a period of ~81 days. Around 2,516 per day.

Yet between many patches from Sep 25 2013 to May 16 2014, the version numbers only rose 305,922 over a period of ~233 days, almost 3x the time between recent versions. It only rose an average of ~1,313 per day.

Without a linear timescale, it's very hard to compare versions, especially ones with 7 digits. If we could go back to the old version number method, days since Nov 7, 1998, it would be much more manageable. Barring that, at least a linear timescale would be nice.

The number comes from steam.inf and to my knowledge the code for the generation of that isn't open source?

Wherever the code that generates steam.inf is would need to be changed, or just change the `version` command code to go off of a linear number.

**Comment by Thespikedballofdoom** [comment]
ah, well good to know I guess. Really is a strange scheme
